include/swift/AST/DiagnosticsCommon.def

```diff
@@ -32,6 +32,8 @@ ERROR(error_opening_output,none,
 ERROR(error_closing_output,none,
       "error closing '%0' for output: %1", (StringRef, StringRef))
 
+ERROR(error_mccas, none, "%0", (StringRef))
```

Comment:
Please write a more elaborate error message.

Reply:
Sorry about that